admin_ui/src/App.vue

```diff
@@ -121,6 +120,14 @@ export default Vue.extend({
             }
         )
 
+        axios.interceptors.request.use((config) => {
+            const languages = JSON.parse(localStorage.getItem("languages"))
+            for (const lang in languages) {
+                this.$i18n.setLocaleMessage(lang, languages[lang])
+            }
+            return config
+        })
```

Comment:
I'm not sure about this - it will trigger on every axios request. Better just to fetch the languages and then store them once.